app/dashboard/views.py

```diff
@@ -1006,6 +1006,34 @@ def bounty_details(request, ghuser='', ghrepo='', ghissue=0, stdbounties_id=None
     return TemplateResponse(request, 'bounty/details.html', params)
 
 
+def get_notify_funder_modal(request, stdbounties_id):
+    stdbounties_id = clean_str(stdbounties_id)
+    bounty = Bounty.objects.filter(standard_bounties_id=stdbounties_id).first()
+
+    context = {
+        'bounty': bounty,
+        'active': 'get_notify_funder_modal',
+        'title': _('Send Payout Reminder')
+    }
+    return TemplateResponse(request, 'notifyfunder.html', context)
+
+
+@csrf_exempt
+def funder_payout_reminder(request, bounty_id):
```

Comment:
Same as above ^.
Also you need to check that the user is authenticated first and allowed to notify the funder 👍